spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/SparkWriteConf.java

```diff
@@ -244,4 +245,18 @@ public boolean useTableDistributionAndOrdering() {
         .defaultValue(SparkWriteOptions.USE_TABLE_DISTRIBUTION_AND_ORDERING_DEFAULT)
         .parse();
   }
+
+  public Long validateFromSnapshotId() {
+    return confParser.longConf()
+        .option(SparkWriteOptions.VALIDATE_FROM_SNAPSHOT_ID)
+        .parseOptional();
+  }
+
+  public IsolationLevel dynamicOverwriteIsolationLevel() {
```

Comment:
Based on the discussion we had on isolation levels, I think this the default should be `null` (i.e. no validation). Serializable isolation should mean validate both new data files and all types of deletes. Snapshot isolation would mean validate just all types of deletes.

Reply:
Yep, done.